tasks/eth/fp-recovery/003-enable-permissionless-game/input-template.json

```diff
@@ -0,0 +1,38 @@
+{
+  "chainId": 1,
+  "metadata": {
+    "name": "Deputy Guardian - Enable Permissioness Dispute Game",
+    "description": "This task updates the `respectedGameType` in the `OptimismPortal` to `CANNON`, enabling users to permissionlessly propose outputs as well as for anyone to participate in the dispute of these proposals. This action requires all in-progress withdrawals to be re-proven against a new `FaultDisputeGame` that was created after this update occurs."
+  },
+  "transactions": [
+    {
+      "metadata": {
+        "name": "Update `respectedGameType` in the `OptimismPortal`",
+        "description": "Updates the `respectedGameType` to `CANNON` in the `OptimismPortal`, enabling permissionless proposals and challenging."
+      },
+      "to": "0xc6901F65369FC59fC1B4D6D6bE7A2318Ff38dB5B",
+      "value": "0x0",
+      "data": "0xa1155ed900000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "contractMethod": {
+        "type": "function",
+        "name": "setRespectedGameType",
+        "inputs": [
+          {
+            "name": "_portal",
+            "type": "address"
+          },
+          {
+            "name": "_gameType",
+            "type": "uint32"
+          }
+        ],
+        "outputs": [],
+        "stateMutability": "nonpayable"
+      },
+      "contractInputsValues": {
+        "_gameType": "0",
+        "_portal": "0x0000000000000000000000000000000000000000"
```

Comment:
NIT: Swapping both to keep the same signatures order is better for readibility ->  `setRespectedGameType(address,uint32)`.